.dockerignore

```diff
@@ -0,0 +1,19 @@
+# Directories
+service-worker
+src
+static
+.git
+
+# Files
+*.md
+*.txt
+*.png
+Makefile
+package.json
+build-babel-preset.js
+build-pm2-processes.js
+yarn.lock
+
+# Binaries
```

Comment:
remove `# Binaries` ?

Reply:
This should be executables. But I was wondering why these two files are executable.